boards/trellis_m4/Cargo.toml

```diff
@@ -41,6 +42,7 @@ ws2812-nop-samd51 = { git = "https://github.com/smart-leds-rs/ws2812-nop-samd51.
 default = ["rt", "atsamd-hal/samd51g19a", "atsamd-hal/samd51"]
 rt = ["cortex-m-rt", "atsamd-hal/samd51g19a-rt"]
 unproven = ["atsamd-hal/unproven"]
+keypad-unproven = ["keypad", "unproven"]
```

Comment:
I would just call this keypad

Reply:
The problem is crate dependencies and cargo features exist in the same namespace, so I can't have `keypad` activate the `unproven` feature. That said, the existing gating can be changed from:

```
#[cfg(feature = "keypad-unproven"))]
```

to

```
#[cfg(all(feature = "keypad", feature = "unproven"))]
```